client/components/resources/resource-reviews.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {ReviewCard} from '../reviews'
+import {getAllReviewsOfResource} from '../../store'
+
+const ResourceReviews = (props) => {
+  const reviews = props.reviews[props.resourceName]
+  return(
+    <div>
+      { reviews && <h3> Top Reviews: </h3> }
```

Comment:
If there are no reviews for a resource, it returns an empty object, which is truthy.  Therefore, this will always render.

Reply:
This  doesn't seem to be a problem when Im running it.  Probably because reviews are coming from line 7 props.reviews[props.resourceName] , and not an empty object.